src/js_schema.hpp

```diff
@@ -85,6 +85,13 @@ static inline void parse_property_type(StringData object_name, Property& prop, S
         type = type.substr(0, type.size() - 1);
     }
 
+    DictionarySchema dictionary {type};
+
+    if(dictionary.is_dictionary()){
```

Comment:
Use spacing similar to the rest of the code. After "if", before "{".

Reply:
Please ignore code formatting - we plan to apply `clang-format` using Realm Core's configuration when we merge `develop` to `master` (most likely: immediately after the merge).